src/pages/products/Products.jsx

```diff
@@ -286,13 +286,9 @@ const Products = () => {
         </div>
         <div className="product-content-card-section pd-bottom-4x">
           {productData.map((item) => (
-            <Link
-              to={`/product/${item._id}`}
-              className="no-deco product-card"
-              key={item._id}
-            >
+            <div className="product-card" key={item._id}>
               <ProductCard data={item} navigate={navigate} />
```

Comment:
You don't need to pass navigate function here. Wherever you require it just import it there. It is a context.

Reply:
Ok, will do that